src/Manager/nrfutil/west.ts

```diff
@@ -52,8 +55,12 @@ const west = (
 
         tcm.stderr.on('data', err => logger.debug(err));
         tcm.stdout.on('data', data => {
-            logger.debug(data.toString().trimEnd());
-            onUpdate(data);
+            const strippedLog = stripAndPrintNrfutilLogOutput(data.toString());
+
+            if (strippedLog?.length > 0) {
+                logger.debug(strippedLog.trimEnd());
+                onUpdate(strippedLog);
+            }
```

Comment:
When using `--json` it seems a bit inconsistent, that `nrfutil-toolchain-manager launch` still copies the output of the command verbatim to STDOUT instead of wrapping it into JSON, doesn't it? But I guess for now this is fine. Maybe in the future we can request that system changes this.